src/components/Person/Person.jsx

```diff
@@ -1 +1,26 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => (
+  <section className="Person">
+    <h2 className="Person__name">
+      {`My name is ${person.name}`}
+    </h2>
+
+    {person.age !== undefined && (
+      <p className="Person__age">{`I am ${person.age}`}</p>
+    )}
+
+    {person.isMarried ? (
+      <>
+        <p className="Person__partner">
+          {`${person.partnerName} is my `}
+          {person.sex === 'm' ? (
+            'wife'
+          ) : (
+            'husband'
+          )}
```

Comment:
For bettter readability this logic should be moved into variable
`const partner = person.sex === 'm' ? 'wife' : 'husband';`
```suggestion
          {`${person.partnerName} is my ${partner }`}
```